src/worker/schema/migrate-None-to-0.sql.jinja

```diff
@@ -0,0 +1,71 @@
+-- WARN all '<user>'@'%' must be removed in production
+
+CREATE USER IF NOT EXISTS 'qsmaster'@'localhost';
+CREATE USER IF NOT EXISTS 'qsmaster'@'%';
+GRANT ALL ON `q\_memoryLockDb`.* TO 'qsmaster'@'localhost';
```

Comment:
MySQL accounts and the corresponidng (optional) passwords need to be injected into this file like it's done in `src/admin/schema/migrate-None-to-0.sql.jinja`. In general, anything that's either site-specific or security-related has to be injected into the database initialization sequences.